### PR TITLE
Clicking on bug link open native bug page

### LINKED ISSUE #110: [Github] Clicking on bug link should open bug page
Currently bug links which are shown in pull requests like #108 etc, are hyperlinks i.e they open in a browser when you click them. However why can't we instead take the bug number and use that information to open the respective issue page in Project Dashboard itself instead of showing a web page? This will make it a more pleasant user experience.

### PR DESCRIPTION
**Pull request by krnekhelesh**
Fixes #110

### CODE REVIEW

**Comment by krnekhelesh** [comment]
hmm, it needs a bit more testing...do not review yet

**Comment by krnekhelesh** [comment]
Nevermind it works :)...I was testing with an older version :P

File: plugins/github/IssuePage.qml [hunk 1]
```diff
@@ -339,8 +339,23 @@ Page {
                     anchors.centerIn: parent
                     textFormat: Text.RichText
 
+                    property Issue linkIssue
```
Comment (iBelieve) [inline plugins/github/IssuePage.qml]: Is there a reason this is needed? Or could it be a local var inside `onLinkActivated`?

**Comment by iBelieve** [comment]
Looks good!

**Pull request merged**